Project.toml

```diff
@@ -1,7 +1,10 @@
-name = "Assignment07"
+name = "BISC195Bioinformatics"
 uuid = "67893044-1481-44cf-8ab6-de350e09ee4e"
-authors = ["Kevin Bonham, PhD <kbonham@wellesley.edu>"]
+authors = ["Kevin Bonham, PhD <kbonham@wellesley.edu>", "Jessica Maciuch, The Magnificent"]
 version = "0.1.0"
 
+[deps]
+BioSequences = "7e6ae17a-c86d-528c-b3b9-7f778a29fe59"
```

Comment:
Need to do `] add Statistics` - for package code, packages in the StdLib need to be explicitly listed as dependencies.